### PR TITLE
fix padding on icon in deployment modal (#4343)

### PR DESCRIPTION
**Pull request by tehnuge**
## What kind of change does this PR introduce?

Bug fix for #4343
https://github.com/codesandbox/codesandbox-client/issues/4343
## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Click 'Deploy with Vercel' and verify padding:
image

## Checklist

- [ ] Documentation
- [ x] Testing 
- [ x] Ready to be merged
      
- [ x] Added myself to contributors table

### CODE REVIEW

**Comment by CompuIves** [comment]
Padding on the left looks great!! Could you also make sure that the "Please Sign In" doesn't have a linebreak? Now come to think of it, we can actually re-use the component we show here:

image

It's defined here: `packages/app/src/app/pages/common/VercelIntegration/index.tsx`

File: packages/app/src/app/pages/common/Modals/DeploymentModal/index.tsx [hunk 1]
```diff
@@ -63,7 +63,11 @@ export const DeploymentModal: FunctionComponent = () => {
             borderRadius: 'medium',
           })}
         >
-          <VercelIcon />
+          <VercelIcon
+            css={css({
+              padding: '8px',
+            })}
+          />
```
Comment (MichaelDeBoey) [inline packages/app/src/app/pages/common/Modals/DeploymentModal/index.tsx:70]:
```suggestion
          <VercelIcon css={css({ padding: '8px' })} />
```

**Comment by tehnuge** [comment]
@CompuIves I re-used the VercelIntegration component instead. However it looks like it's failing the ES module "is fast" test. Any input on that?

**Comment by CompuIves** [comment]
Ahh, this is great!! Thank you so much, I have one last nitpick. Could you remove the `small` prop from the Integration, and then increase the width of the deployment modal to 600? You can do that here: https://github.com/codesandbox/codesandbox-client/blob/master/packages/app/src/app/pages/common/Modals/index.tsx#L56-L59.

This way it's 100% consistent with the one in the preferences. Other than that it looks perfect!

> However it looks like it's failing the ES module "is fast" test.

A yes, this is a flaky test. I'm going to disable that one.

**Comment by tehnuge** [comment]
@CompuIves Yup, that's done. I also updated small as an optional prop.

**Comment by CompuIves** [comment]
This is beautiful, thanks for the high quality PR!

**Pull request merged**